File: MinAvgTwoSlice/ruby/TestMinAvgTwoSlice.rb
require_relative 'MinAvgTwoSlice'
require 'minitest/autorun'

class Test < MiniTest::Test

  def setup
    @min_avg_two_slice = MinAvgTwoSlice.new
  end

  def test_solution
    assert_equal 1, @min_avg_two_slice.solution([4,2,2,5,1,5,8])
  end

end

File: MinAvgTwoSlice/php/TestMinAvgTwoSlice.php
<?php
require 'MinAvgTwoSlice.php';

class Test extends PHPUnit_Framework_TestCase {

	protected $min_avg_two_slice;

	protected function setup() {
		$this->min_avg_two_slice = new \Codility\MinAvgTwoSlice;
	}

	public function testSolution() {
		$this->assertEquals(1, $this->min_avg_two_slice->solution([4,2,2,5,1,5,8]));
	}
}